### DISCUSSION TITLE
Bundled codes

### DISCUSSION THREAD

**Discussion by jleberre10** [CMSgov/price-transparency-guide#449] [Q&A] [answered]
This question is about how to properly represent bundled codes.

Let's say for some plan, codes 11,12, and 13 are bundled together for 50 dollars. That is to say, if **any combination** of those 3 codes shows up on a claim, the price for all of them combined will be 50 dollars.
Does each code need its own negotiated rate object, with the other codes listed under the bundled codes section?
Or can there be 1 negotiated rate object to represent the bundle?

The example seems to indicate that only 1 rate object is needed, which makes sense since there is only one price associated with the codes, but it's not clear whether that representation means the same thing as the scenario I explained in this case.

**Comment by shaselton-usds** [comment, marked as answer]
@jleberre10 You're correct. There would be the single `negotiated_prices` object associated with the appropriate `provider_groups` for the bundle. Then all of the items/services that are included in that negotiated bundle would be included in the `bundled_codes` array.

There might be multiple `negotiated_prices` objects for a single bundle when the negotiation/price is different for different provider groups.

**Reply by v-sabnani** [reply]
In this example above, what'd we populate in the billing_code field that is outside the bundled_codes array?

**Reply by v-sabnani** [reply]
@shaselton-usds thoughts on the above?
